src/main/java/frc/utils/Vector.java

```diff
@@ -5,6 +5,8 @@
 // with operations for manipulating them
 
 public class Vector {
+  public static final Vector Origin = new Vector();
```

Comment:
nit: prefer the explicit constructor here so we don't have to depend on the implementation of the default constructor (which could change at any time).

Reply:
since I made the default constructor, I know it will always be (0, 0), but I can make it more explicit

Reply:
I think it'd be better to use 0, 0 to make it more futureproof since we plan on reusing this code.